tests/rados/test_bluestore_features.py

```diff
@@ -96,6 +98,39 @@ def run(ceph_cluster, **kw):
 
     rados_object.change_heap_profiler_state(osd_list, "stop")
 
+    execution_time = config["execution-time"]
```

Comment:
Add reasoning behind setting the scrub params... Since we are not using the start_scrub and deep_scrub methods.